libc/include/sys/time.yaml

```
@@ -5,5 +5,10 @@ macros: []
 types:
   - type_name: struct_timeval
 enums: []
-functions: []
 objects: []
+functions:
+  - name: utimes
+    return_type: int
+    arguments:
+      - type: ptr<const char>
+      - type: ptr<const struct timeval>
```

The format is the same as how the type is written in C.
```suggestion
      - type: const char*
      - type: const struct timeval*
```